src/oclif/hooks/utils.ts

```diff
@@ -36,10 +36,13 @@ export function renderCommands(commandsId: Record<number, string>, groups: Comma
   body.push(help.root())
   body.push(' ')
 
-  for (const [key, value] of Object.entries(commandsId)) {
-    body.push(chalk.bold(value))
-    body.push(indent(renderCommand(groups[key !== '255' ? key : commandsGroupLength - 1], ctx), 2))
-    body.push(' ')
+  for (let [key, value] of Object.entries(commandsId)) {
+    key = key !== '255' ? key : (commandsGroupLength - 1).toString()
+    if (groups[key].length) {
```

Comment:
Please test explicity that the list length is greater than 0. Also This conditional is too optimistic. Maybe test first if `group[key]` is indeed different than `undefined`.

Reply:
You’re right!

Reply:
But just to make things clear, `group[key]` will never be `undefined` since this array is inittialised with `[]` 

https://github.com/vtex/toolbelt/blob/d7336a5c09c8c50f25438e27af9c79c2250513aa/src/oclif/hooks/init.ts#L235